flink-runtime/src/test/java/org/apache/flink/runtime/zookeeper/ZooKeeperStateHandleStoreTest.java

```diff
@@ -271,7 +268,7 @@ public void testRepeatableCleanupWithLockOnNode() throws Exception {
         assertEquals(
                 "discardState shouldn't have been called, yet.",
                 0,
-                stateHandle.getNumberOfDiscardCalls());
+                stateHandle.getNumberOfSuccessfulDiscardCalls());
```

Comment:
I guess, here we can even check `stateHandle.getNumberOfDiscardCalls()`, because we're not expecting `discardState` to be called at all, yet